docs/dashboards/scheduled-report.md

```diff
@@ -20,45 +20,74 @@ If a .pdf and/or .png export fails, an error email will be sent to your email ID
+1. Navigate to the dashboard, then click the kebab menu and select **Edit Reports**.
+1. Click the kebab menu and select **Edit** against the required report. <br/><img src={useBaseUrl('img/dashboards-new/scheduled_reports/update_scheduled_report_1.png')} alt="update_scheduled_report" style={{border: '1px solid gray'}} width="600"/>
+1. Make your desired changes. When you're done, click **Update**. <br/><img src={useBaseUrl('img/dashboards-new/scheduled_reports/update_scheduled_report.png')} alt="update_scheduled_report" style={{border: '1px solid gray'}} width="600"/>
+
+:::note
+- Toggle ON or OFF the status button to activate or deactivate the scheduled report.
+:::
 
 ## Delete a scheduled report
 
 To cancel a scheduled report:
 
-1. Navigate to the dashboard, then click the kebab menu and select **Update Scheduled Report**.
-1. Click the down arrow next to the **Update** button, then click **Delete Report**. <br/><img src={useBaseUrl('img/dashboards-new/scheduled_reports/delete_scheduled_report.png')} alt="Delete Report button" width="250"/>
+1. Navigate to the dashboard, then click the kebab menu and select **Edit Reports**.
+1. Click the kebab menu and select **Delete** against the required report.<br/><img src={useBaseUrl('img/dashboards-new/scheduled_reports/delete_scheduled_report_1.png')} alt="delete_scheduled_report" style={{border: '1px solid gray'}} width="600"/>
+1. In the popup window, click **Delete**. <br/><img src={useBaseUrl('img/dashboards-new/scheduled_reports/delete_scheduled_report.png')} alt="delete_scheduled_report" style={{border: '1px solid gray'}} width="500"/>
+
+OR
+
+1. Navigate to the dashboard, then click the kebab menu and select **Edit Reports**.
+1. Click the kebab menu and select **Edit** against the required report. <br/><img src={useBaseUrl('img/dashboards-new/scheduled_reports/update_scheduled_report_1.png')} alt="update_scheduled_report" style={{border: '1px solid gray'}} width="600"/>
+1. Click the down arrow next to the **Update** button, then click **Delete Report**. <br/><img src={useBaseUrl('img/dashboards-new/scheduled_reports/delete_scheduled_report_2.png')} alt="delete_scheduled_report" style={{border: '1px solid gray'}} width="600"/>
+
+## Duplicate a Scheduled Report
+
+To duplicate to an existing a Scheduled Report:
+
+1. Navigate to the dashboard, then click the kebab menu and select **Edit Reports**.
+1. Click the kebab menu and select **Duplicate** against the required report.<br/><img src={useBaseUrl('img/dashboards-new/scheduled_reports/duplicate_scheduled_report.png')} alt="update_scheduled_report" style={{border: '1px solid gray'}} width="600"/>
+1. Make your desired changes. When you're done, click **Update**. <br/><img src={useBaseUrl('img/dashboards-new/scheduled_reports/update_scheduled_report.png')} alt="update_scheduled_report" style={{border: '1px solid gray'}} width="600"/>
+
+## Subscribe to a Scheduled Report
```

Comment:
One thing I think worth mentioning is the unsubscribe functionality, specifically that you will not be able to unsubscribe if you are the only recipient.

Not sure if that would be in this section or deserve it's own.